frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/embedding/enterprise.embedding.unit.spec.tsx

```diff
@@ -109,23 +124,76 @@ describe("[EE, no token] embedding settings", () => {
       });
     });
   });
+
+  it("should link to quickstart for interactive embedding", async () => {
+    await setupEnterprise({
+      settingValues: {
+        "enable-embedding": false,
+        version: { tag: "v1.49.3" },
+      },
+    });
+    expect(getQuickStartLink()).toBeInTheDocument();
+    expect(getQuickStartLink()).toHaveProperty(
+      "href",
+      "https://www.metabase.com/docs/v0.49/embedding/interactive-embedding-quick-start-guide.html?utm_source=oss&utm_media=embed-settings",
+    );
```

Comment:
I suppose the behaviour for quickstart link is the same for both OSS and EE, just with a different version tag?

Reply:
It seems the only differences are UTM tags.

You can see that the version is still the same for EE users
https://github.com/metabase/metabase/blob/71f208f11a2df3dd82bd3f53305bf86eb080af00/frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/embedding/premium.embedding.unit.spec.tsx#L83

Reply:
@WiNloSt I see! I didn't spot the `utm_source` differences at first glance.